### DISCUSSION TITLE
Navigate interface with keyboard

### DISCUSSION THREAD

**Discussion by ghost** [IndiePython/nodezator#83] [Feature Request]
- Enter to "Ok" notices
- Node placement, movement (arrows do work already)
- Node connection
- ...

**Comment by KennedyRichard** [comment]
Thank you for the ideas.

> Enter to "Ok" notices

This one has been suggested before, I already included a flag that makes it so dialogues can be dismissed with the keyboard, but I have yet to visit each dialogue in the source to turn the flag on for the dialogues that can be dismissed in that way, since some dialogues cannot be dismissed or accepted in that way because the right choice to pick is not immediately obvious. Of course, when the prompt is merely to "Ok" an information, as you pointed out, I'll turn this option on so pressing "Enter" can dismiss it right away as well.

> Node placement, movement (arrows do work already); Node connection

Could you elaborate a bit more on these options, please? How do you expect these elements to behave based on the keyboard usage, and which keys would be involved?

**Reply by GrimPixel** [reply]
I think it can be like Vim. For example, a key and a number means going to the node with the number.

**Reply by KennedyRichard** [reply]
> I think it can be like Vim. For example, a key and a number means going to the node with the number.

Yes, seems like a great idea!
